bbtools/38.96/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:xenial
+
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="BBTools"
+LABEL software.version="38.96"
+LABEL description="A set of tools labeled as \"Bestus Bioinformaticus\""
+LABEL website="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/"
+LABEL license="https://jgi.doe.gov/disclaimer/"
+LABEL maintainer="Abigail Shockey"
+LABEL maintainer.email="abigail.shockey@slh.wisc.edu"
+LABEL maintainer2="Padraic Fanning"
+LABEL maintainer2.email="faninnpm AT miamioh DOT edu"
+
+RUN apt-get update && apt-get install -y build-essential \
+    openjdk-8-jre-headless \
+    wget && \
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get autoclean
+
+# get miniconda and the artic-ncov19 repo
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && \
+    bash ./Miniconda3-latest-Linux-x86_64.sh -p /miniconda -b  && \
+    rm Miniconda3-latest-Linux-x86_64.sh
+
+# set the environment
+ENV PATH="/miniconda/bin:$PATH"
+
+RUN conda config --add channels defaults && \
```

Comment:
instead of installing sambamba and samtools from conda, I think it might be better to install these tools from their git repo

You would need to add more tools in your apt-get command, but it would look more like:

```
# install samtools
RUN wget https://github.com/samtools/samtools/releases/download/${SAMTOOLSVER}/samtools-${SAMTOOLSVER}.tar.bz2 && \
 tar -xjf samtools-${SAMTOOLSVER}.tar.bz2 && \
 rm samtools-${SAMTOOLSVER}.tar.bz2 && \
 cd samtools-${SAMTOOLSVER} && \
 ./configure && \
 make && \
 make install
 ```

Reply:
On a related note, we don't have a standalone container for sambamba. 

I feel like that should be remedied. Eventually.

Reply:
Here's a samtools dockerfile that you can use for reference : https://github.com/StaPH-B/docker-builds/blob/master/samtools/1.14/Dockerfile

Reply:
Incidentally, [a new version of samtools was released recently](https://github.com/samtools/samtools/releases/tag/1.15).

Note to self: per the Bioconda recipes, we need the following:
- `libcurl`
- `bzip2`
- `xz`
- `zlib`
- `libdeflate`
- `openssl`
- `ncurses`

Reply:
I noticed the samtools update this morning. I'll be updating samtools/bcftools/htslib next week (unless someone gets to it sooner)